pkg/plugin/trivy/plugin.go

```diff
@@ -862,6 +909,212 @@ func (p *plugin) getPodSpecForClientServerMode(ctx starboard.PluginContext, conf
 	}, secrets, nil
 }
 
+//FileSystem scan option with standalone mode.
+//The only difference is that instead of scanning the resource by name,
+//We scanning the resource place on a specific file system location using the following command.
+//
+//     trivy --quiet fs  --format json --ignore-unfixed  file/system/location
+//
+func (p *plugin) getPodSpecForStandaloneFSMode(ctx starboard.PluginContext, config Config,
+	workload client.Object) (corev1.PodSpec, []*corev1.Secret, error) {
+	var secrets []*corev1.Secret
+	spec, err := kube.GetPodSpec(workload)
+	if err != nil {
+		return corev1.PodSpec{}, nil, err
+	}
+	// get nodeName from running pods.
+	spec.NodeName, err = p.objectResolver.GetNodeName(context.Background(), workload)
```

Comment:
I think we should reuse context.Context passed from Starboard to the plugin instead of instantiating a new one.